doc/api/fs.md

```diff
@@ -1640,7 +1640,9 @@ Read data from the file specified by `fd`.
 `length` is an integer specifying the number of bytes to read.
 
 `position` is an integer specifying where to begin reading from in the file.
-If `position` is `null`, data will be read from the current file position.
+If `position` is `null`, data will be read from the current file position, 
+and the file position will be updated for subsequent reads.
+If `position` is an integer, the file position will remain unchanged.
```

Comment:
yeah and let's say what happen if you have a position that goes over the size of the file and specify that it's bytes, also say that the position in the end is the quantity that has been read + the previous position but let's wait that you do it on an other PR because this has noting to do with this one